packages/grafana-ui/src/components/Button/Button.story.tsx

```diff
@@ -1,25 +1,41 @@
-import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { ComponentStory } from '@storybook/react';
 import React from 'react';
 
 import { ComponentSize } from '../../types/size';
 import { Card } from '../Card/Card';
 import { HorizontalGroup, VerticalGroup } from '../Layout/Layout';
 
-import { allButtonVariants, allButtonFills, Button } from './Button';
+import { allButtonVariants, allButtonFills, Button, ButtonProps } from './Button';
 import mdx from './Button.mdx';
 import { ButtonGroup } from './ButtonGroup';
 
-const meta: ComponentMeta<typeof Button> = {
+export default {
   title: 'Buttons/Button',
   component: Button,
   parameters: {
+    controls: { expanded: true },
     docs: {
       page: mdx,
     },
   },
+  argTypes: {
+    size: {
+      options: ['sm', 'md', 'lg'],
```

Comment:
nit: Not a biggie but we could pull `sizes` out of `Examples` story and reference it here.

Reply:
Done